invenio_collections/__init__.py

```diff
@@ -10,6 +10,6 @@
 
 from .ext import InvenioCollections
 
-__version__ = "0.1.0"
+__version__ = "1.0.0"
```

Comment:
We can either release `1.0.0` or `0.4.0`, I don't have a preference. I don't think we need to go as far as releasing `2.0.0` since all the v1 releases were alpha ones.